frontend/src/components/widgets/buttons/TextButton.svelte

```diff
@@ -15,6 +15,7 @@
 	export let icon: IconName | undefined = undefined;
 	export let hoverIcon: IconName | undefined = undefined;
 	export let emphasized = false;
+	export let focused = false;
```

Comment:
Changing this file shouldn't be necessary. You can just use the existing `emphasized`.

Reply:
I suggested using the hover colors for keyboard selection, white looks to harsh in my opinion. Makes it consistent with mouse input.

Reply:
CSS in the dialog would be acceptable.